workloads/burn-in/install_hplbit_deps.sh

```diff
@@ -0,0 +1,239 @@
+#!/bin/bash 
+#Install dependencies for the HPL Burn-In Test.  It is assumed that Slurm Cluster Manger is configured with PMIx and hwloc. This script specifically installs the CUDA Toolkit by its .run installer, and UCX and OpenMPI via the HPC-X Software Toolkit. 
+
+
+#Usage instructions for build dependencies script
+print_usage() {
+   cat << EOF
+
+${0} [options]
+
+Select the HPL Burn In Test Script dependencies to be built for the cluster under test. 
+
+When ${0} is run with no options, it defaults to installing:
+CUDA Toolkit 11.0.2
+HPC-X v2.7 which install UCX 1.9 and Open MPI 4.0.4
+
+Options:
+    -c|--cuda <CUDA Toolkit>
+        * Set to 0 in order to not build the CUDA toolkit.  For a Slurm cluster set up by DeepOps, CUDA is setup by default so only HPC-X needs to be run. 
+    -m|--mofed <Mellanox OFED major version>
+        * Define the MOFED major version to install the correct version of HPC-X for the OFED version on the cluster compute nodes. FOR DGX A100 this is 5.0, DGX-1/2 it is 4.7
+    -u|--ucx <Install OpenMPI and UCX via HPC-X>
+	* Set to 0 in order to not utilize HPC-X Cluster Kit to install OpenMPI and UCX
+EOF
+
+exit
+}
+
+
+
+while [ $# -gt 0 ]; do
+        case "$1" in
+                -h|--help) print_usage ; exit 0 ;;
+                -c|--cuda) buildcuda="$2"; shift 2 ;;
+                -m|--mofed) ofed_major="$2"; shift 2 ;;
+                -u|--ucx) buildhpcx="$2"; shift 2;;  
+		*) echo "Option <$1> Not understood" ; exit 1 ;;
+
+        esac
+done
+
+#Next steps after CUDA build option selected and MOFED Specified
```

Comment:
In most of the DeepOps scripts our comment style has a space after the # followed by a capital letter

Reply:
Got it.  Will follow convention in the future.